content/tracing/getting_further/trace_sampling_and_storage.md

```diff
@@ -67,19 +85,19 @@ Each trace has a `sampling_priority`, which is assigned at its inception and pro
 
-**Note**: Spans dropped by priority sampler can still be kept by the signature sampler. The backend can re-sample to keep up to 60 traces per minute per host. The distributed traces that are kept will all be complete. 
+**Note**: Spans dropped by priority sampler can still be sampled by the signature sampler. The backend can re-sample to keep up to 60 traces per minute per host. The distributed traces that are kept will all be complete. 
 
 #### Using Priority Sampling
 
-Priority Sampling is recommended if your traces are distributed across multiple hosts or if you need finer control over the traces sampled. To use it, enable `distributed_sampling` in your client ([Ruby](http://www.rubydoc.info/gems/ddtrace/#Priority_sampling), [Python](http://pypi.datadoghq.com/trace/docs/#priority-sampling), [Go](https://godoc.org/github.com/DataDog/dd-trace-go/tracer#Span.SetSamplingPriority), [Java](/tracing/languages/java/#sampling-distributed-tracing)) as it is disabled by default.
+Priority Sampling is recommended if your traces are distributed across multiple hosts or if you need finer control over the traces sampled. To use it, enable `distributed_sampling` in your client ([Ruby](http://www.rubydoc.info/gems/ddtrace/#Priority_sampling), [Python](http://pypi.datadoghq.com/trace/docs/#priority-sampling), [Go](https://godoc.org/github.com/DataDog/dd-trace-go/tracer#Span.SetSamplingPriority), [Java](/tracing/setup/java/#sampling-distributed-tracing)) as it is disabled by default.
 
 ### Client rate sampling
 
-It is possible to disable the instrumentation for a percentage of transactions. Datadog Tracers are performant and can run with a minimal cost with thousands of requests per second, however client rate sampling can be used to further reduce the instrumentation footprint. In this case, no trace will be generated at all, and Datadog will estimates aggregate statistics (requests per second, error rate, etc.). An example of this in Python can be found here: http://pypi.datadoghq.com/trace/docs/#pre-sampling.
+It is possible to disable the instrumentation for a percentage of transactions. Datadog Tracers are performant and can run with a minimal cost with thousands of requests per second, however client rate sampling can be used to further reduce the instrumentation footprint. In this case, no trace will be generated at all, and Datadog estimates aggregate statistics (requests per second, error rate, etc.). An example of this in Python can be found here: http://pypi.datadoghq.com/trace/docs/#pre-sampling.
 
 ## Trace storage
 
-Individual traces are retained in the Datadog backend for up to 6 months, according to the following time buckets:
+Individual traces are stored for up to 6 months but to determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
```

Comment:
I would break this up into multiple sentences:

> Individual traces are stored for up to 6 months but to determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime.

becomes:

> Individual traces are stored for up to 6 months. To determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime.

Reply:
👌